docs/embrace-api/metrics-migration.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 5
 
 # Metrics migration (2023-10-23)
 
-Embrace is updating its beta Metrics API to support interoperability.  This means all metrics, both the pre-built or "Standard" and Custom Metrics, can be combined in any analyses you plan on doing.  Sum across time-ranges and other dimensions, compare one Metric with another, or use multiple metrics to compute relevant rates.
+Embrace is updating its beta Metrics API to support interoperability.  This means all metrics, both the pre-built or Standard and Custom Metrics, can be combined in any analyses you plan on doing.  Sum across time-ranges and other dimensions, compare one Metric with another, or use multiple metrics to compute relevant rates.
```

Comment:
> This means all metrics, both the pre-built or Standard and Custom Metrics, can be combined in any analyses you plan on doing.

Can you make this sentence "This means all metrics, both the pre-built Standard Metrics and your Custom Metrics, can be combined in any analyses you plan on doing.